RELEASE-NOTES.md

```diff
@@ -1,5 +1,61 @@
 # Prefect Release Notes
 
+## Release 2.11.5
+
+### New Guides
+
+We're happy to announce two new guides aimed at helping users get the most out of their Prefect setup!
+
+#### How to move data to and from cloud providers
+
+Moving data to cloud-based storage and retrieving it is crucial in many data engineering setups. This guide provides step-by-step instructions to seamlessly integrate and interact with popular cloud services like AWS, Azure, and GCP using Prefect.
+
+#### Running flows with Kubernetes
+
+For those aiming to optimize their flows using Kubernetes, this guide provides a deep dive on how to efficiently run flows on Kubernetes using containers. Catering to both novices and seasoned experts, the content offers insights for all proficiency levels.
```

Comment:
It might be nice to link to these guides in the docs

Reply:
Do we know what the URL will be since they haven't been published yet?

Reply:
+1

Reply:
Yes, I proposed revisions to include them